src/modules/components/proposal/proposalActions/actions/proposalActionChangeMembers/proposalActionChangeMembers.test.tsx

```diff
@@ -0,0 +1,102 @@
+import { render, screen } from '@testing-library/react';
+import { modulesCopy } from '../../../../../assets';
+import { ProposalActionType, type IProposalActionChangeMembers } from '../../proposalActionsTypes';
+import { generateProposalActionChangeMembers } from '../generators/proposalActionChangeMembers';
+import { ProposalActionChangeMembers } from './proposalActionChangeMembers';
+
+jest.mock('../../../../../../core', () => ({
+    DataList: {
+        Root: jest.fn(({ children }) => <div>{children}</div>),
+        Container: jest.fn(({ children }) => <div>{children}</div>),
+    },
+    Heading: jest.fn(({ children }) => <h1>{children}</h1>),
+}));
+
+jest.mock('../../../../member', () => ({
+    MemberDataListItem: {
+        Structure: jest.fn(({ address, ensName }) => (
+            <div>
+                <span>{address}</span>
+                <span>{ensName}</span>
+            </div>
+        )),
+        Skeleton: jest.fn(() => <div>Loading...</div>),
+    },
+}));
+
+describe('<ProposalActionAdjustMemberCount /> component', () => {
+    const createTestComponent = (props?: Partial<IProposalActionChangeMembers>) => {
```

Comment:
We should use the props here instead of the action so we can customise the component later on if we add other props
```suggestion
    const createTestComponent = (props?: Partial<IProposalActionChangeMembersProps>) => {
```